### DISCUSSION TITLE
Hi Api Platform Team! I think I found a color constrast problem in minikube code snippets of the documentation

### DISCUSSION THREAD

**Discussion by alexandreelise** [api-platform/api-platform#2617] [General]
Hi,
It seems that a color constrast in the light theme prevents users from seeing the code snippets content of the minikube documentation of api platform.

Here is the screenshot

![screenshot-minikube-api-platform-code-snippets-contrast-problem-2024-01-12](https://github.com/api-platform/api-platform/assets/51425450/2e66c5ef-b50d-447a-8786-7beb05c119dd)

Might want to look at it @dunglas @soyuka 

Enjoy your weekend

**Comment by beejaz** [comment]
I added missing `console` styling in the docs with a PR here https://github.com/api-platform/docs/pull/1874
